### ISSUE TITLE
create an icon for Address

### ISSUE THREAD

**Issue by msayanece** [Templarian/MaterialDesign#2353]

**Comment by MrGrigri** [comment]
Will `map-marker` and/or `account-location` not work for you?

![map-marker](https://materialdesignicons.com/icon/map-marker) ![account-location](https://materialdesignicons.com/icon/account-location)

**Comment by Templarian** [comment]
Adding the alias label.

- [ ] `address-marker`
- [ ] `user-address`
- [ ] `account-address`

**Comment by Templarian** [comment]
Let me know if the suggestion above works. Closing with new aliases, hopefully this helps others searching.

**Comment by msayanece** [comment]
Thaks for your answer. Map-marker is a common icon for location. It should not be used for address. We can use account-location particularly for a person location. Think it this way, address can be for not only person but also car or bus or anything else. Therefore, these two will not work for me. 

after searching with your alias label names, I am unable to find the icons. Please guide me, how to search for it.

Hope, I can clear your doubt.

Here is a sample icon, that is the closest possible solution, I think.

![](https://thumb1.shutterstock.com/display_pic_with_logo/935473/253759033/stock-vector-simple-web-icon-in-vector-address-253759033.jpg)

**Comment by Templarian** [comment]
The icon call sometimes caches (and has a ~15min server cache). Just looked and it is in the search results now.

Reopening the issue to see if any of the contributors have an idea/feedback.

**Comment by msayanece** [comment]
Thanks man. Now I am able to find the results. But, The icon I required should be just like the icon in my previous comment.

**Comment by GreenTurtwig** [comment]
Having a map plus a pin for a single icon won't really be feasible, icons have to recognisable at 1x.  
Also simply adding a map to the design is unnecessary in my opinion, it doesn't represent `address` to me.

If the address is for a home you could always use the `home` icon like Google does in Maps or `home-map-marker`.
[![](https://materialdesignicons.com/icon/home)](https://materialdesignicons.com/icon/home)[![](https://materialdesignicons.com/icon/home-map-marker)](https://materialdesignicons.com/icon/home-map-marker)

**Comment by JamesCoyle** [comment]
Icons in this set are made to be multipurpose and should not be used on their own unless the context of the application makes it obvious what it is used for.  Icons are intended as a quicker way to identify an option than text and work best if the user is already familiar with the application.  As such, any of the icons we have suggested should work fine for your use-case as long as your application is also following the rest of google's guidelines pertaining to accessibility. 

If you need to differentiate between different types of addresses you can always use text to do so. You can also use the `map-marker` icon to denote the section is related to locations and then use `home`, `domain`, `briefcase`, `sign-direction`, `school` etc. to show different types of address.  

To summarize, if your application requires a more specific icon, more often than not it's the application that could do with some adjustments.  Material Design is a complete system that works together to add information by implication while maintaining simplicity and legibility.  Trying to add too much information in one place (such as an icon) can just make things more confusing so always try to think about how you can split things up and add information via context.

**Comment by msayanece** [comment]
`sign-direction` may be one of the work around. But it would be more helpful if anyone can create one like the sample icon.

**Comment by GreenTurtwig** [comment]
As I said before it won't be possible, as it won't be recognisable at 1x and won't follow the Material Design guidelines.

It might be worth closing this issue since, as stated by @JamesCoyle, any of the suggestions will do providing your application has clear labels e.t.c.

**Comment by MrGrigri** [comment]
I second closing this issue.
